python/tk_multi_loader/delegate_publish_list.py

```diff
@@ -8,69 +8,388 @@
+        #  'published_file_type': {'id': 53,
+        #                          'name': 'Flame Render',
+        #                          'type': 'PublishedFileType'},
+        #  'task': None,
+        #  'task.Task.content': None,
+        #  'task.Task.due_date': None,
+        #  'task.Task.sg_status_list': None,
+        #  'task_uniqueness': False,
+        #  'type': 'PublishedFile',
+        #  'version': {'id': 6697,
+        #              'name': 'aaa_00010_F004_C003_0228F8_v000',
+        #              'type': 'Version'},
+        #  'version.Version.sg_status_list': 'rev',
+        #  'version_number': 2}
+        
+        # Publish Name Version 002
+        sg_data = shotgun_model.get_sg_data(model_index)
+        main_text = "<b>%s</b>" % (sg_data.get("name") or "Unnamed")
+        main_text += " Version %03d" % sg_data.get("version_number")        
+
+        # If we are in "show subfolders mode, this line will contain
+        # the entity information (because we are displaying info from several entities 
+        # in a single view. If show subfolders mode is off, the latest description is shown.
+        if self._sub_items_mode:
+            # show items in subfolders mode enabled
+            # get the name of the associated entity
+            
+            main_text += "  ("
+            
+            entity_link = sg_data.get("entity")
+            if entity_link:
+                main_text += "%s <span style='color:#2C93E2'>%s</span>" % (entity_link["type"], entity_link["name"])
+
+            if sg_data.get("task") is not None:
+                main_text += ", Task %s" % sg_data["task"]["name"]
+               
+            main_text += ")"
+        elif sg_data.get("task") is not None:
+            # When not in subfolders mode always show Task info
+            # (similar to the logic in the thumbnail view, but always show)
+            main_text += "  (Task %s)" % sg_data["task"]["name"]
+
+        # Quicktime by John Smith at 2014-02-23 10:34
+        pub_type_str = shotgun_model.get_sanitized_data(model_index, SgLatestPublishModel.PUBLISH_TYPE_NAME_ROLE)            
+        created_unixtime = sg_data.get("created_at") or 0
+        date_str = datetime.datetime.fromtimestamp(created_unixtime).strftime('%Y-%m-%d %H:%M')
+        # created_by is set to None if the user has been deleted.
+        if sg_data.get("created_by") and sg_data["created_by"].get("name"):
+            author_str = sg_data["created_by"].get("name")
+        else:
+            author_str = "Unspecified User"
+        small_text = "<span style='color:#2C93E2'>%s</span> by %s at %s" % (pub_type_str, 
+                                                                            author_str,
+                                                                            date_str)
+        small_text += "<br><b>Description:</b> %s" % sg_data.get("description") or "No description given"
```

Comment:
Looks like some parenthesis are missing here, I'm seeing some "Description: None"

Should be 
`small_text += "<br><b>Description:</b> %s" % (sg_data.get("description") or "No description given")`